docs/source/models.rst

```diff
@@ -1306,37 +1306,42 @@ Deploy a ``python_function`` model on Microsoft Azure ML
 
 The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML, either to Azure Kubernetes Service (AKS) or Azure Container Instances (ACI) for real-time serving. 
 
-The resulting container image contains a web server that accepts the following data formats as input:
+The resulting deployment accepts the following data formats as input:
 
 * JSON-serialized pandas DataFrames in the ``split`` orientation. For example, ``data = pandas_df.to_json(orient='split')``. This format is specified using a ``Content-Type`` request header value of ``application/json``.
 
-Deployments can be generated using both the Python API or MLflow CLI. On both cases, a ``JSON`` configuration file has to be specified with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI). You can check the full specification at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. The deployment configuration file looks as follows:
+Deployments can be generated using both the Python API or MLflow CLI. On both cases, a ``JSON`` configuration file can be indicated with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI) and a minimal configuration. The full specification of this configuration file can be checked at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. On both cases, you will also need the Azure ML MLFlow Tracking URI of your particular Azure ML Workspace where you want to deploy your model. You can obtain this URI in several ways:
 
-.. rubric:: For an ACI deployment
+* In the `Azure ML portal <https://ml.azure.com>`_, in the upper right corner, clic on the name of the workspace and on the pop-up blade that will show-up, clic on ``View all properties in Azure Portal``. In the properties panel, you will see the value corresponding to ``MLflow tracking URI``.
```

Comment:
clic -> click

Also, could you convert this to a list of steps?
```
  * Through the `Azure ML portal <https://ml.azure.com>`:
    * Navigate to your Azure ML portal page.
    * Click on the name of the workspace at the upper right corner of the page.
    * Click "View all properties in Azure Portal" on the pane popup.
    * Copy the ``MLflow tracking URI`` value from the properties panel.

* Programmatically through the use of the ``Azure ML SDK``, using the method ....
```

Reply:
Addressed on [e8f336f](https://github.com/mlflow/mlflow/pull/5491/commits/e8f336f565706d7c1a180afbfd18ebd476c6d2e9)